3711/1/parser/git_hub.rb

```diff
@@ -0,0 +1,78 @@
+require_relative '../ruby_gem'
+require 'json'
+require 'net/http'
+require 'nokogiri'
+require 'open-uri'
+
+module Parser
+  # :reek:TooManyInstanceVariables
+  class GitHub
+    def initialize(gem_name, source_url)
+      @url = source_url
+      @repo = @url[%r{\/[^.:\/]+\/[^.:\/]+}]
+      @gem_data = { 'name' => gem_name, 'url' => @url }
+      @client_id = '26b9a6fc285e836c9f7a'
+      @client_secret = '7ca0b9c0e86047afbac0951d3d505bf7801a5603'
+      @auth_params = "?client_id=#{@client_id}&client_secret=#{@client_secret}"
```

Comment:
This line should be defined locally before executing net http open

Reply:
Made `client_id` and `client_secret` constants and used new method to ser these params to uri